app/controllers/welcome_controller.rb

```diff
@@ -5,9 +5,7 @@ class WelcomeController < ApplicationController
   # Allow iframes on the index URL because safari is an asshole, and doesn't honor our iframe options
   skip_before_filter :set_x_frame_options_header, only: [:bike_creation_graph, :index]
 
-  def index
-    @recovery_displays = RecoveryDisplay.limit(5)
-  end
+  def index; end
```

Comment:
Why leave the empty method definition?

Reply:
Because I like having method definitions for all routes.

Reply:
Rails does it for us! :) But I guess I can see how it can serve as documentation to have the method signature there.